### PR TITLE
Submission

### PR DESCRIPTION
**Pull request by DawidKapka**

### CODE REVIEW

File: functions/src/services/lobby.service.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export class LobbyService {
   }
 
   public async join(user: User, lobby: Lobby) {
-    if (lobby.state == LobbyState.IN_PROGRESS) {
+    if (lobby.state != LobbyState.JOINING) {
```
Comment (DeNic0la) [inline functions/src/services/lobby.service.ts:33]: can you make this !==

File: src/app/storify/game/game.component.html [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 <app-submission
-  *ngIf="gameState === 'submitting' && !loading"
+  *ngIf="gameState === 1 && !loading"
   [lobby]="lobby"
   [story]="story"
   (submit)="submitSentence($event)"
   (tick)="tick($event)"
 ></app-submission>
+<app-submissions-view
+  *ngIf="(gameState === 2 || gameState === 3) && !loading && !submissionsViewed"
```
Comment (DeNic0la) [inline src/app/storify/game/game.component.html:10]: can you make a comment about the state or use enums or something, this is very not readable

File: src/app/storify/game/game.component.html [hunk 3]
```diff
@@ -1,13 +1,19 @@
 <app-loader *ngIf="loading"></app-loader>
 <app-submission
-  *ngIf="gameState === 'submitting' && !loading"
+  *ngIf="gameState === 1 && !loading"
```
Comment (DeNic0la) [inline src/app/storify/game/game.component.html:3]: comment or enum pls

File: functions/src/services/lobby.service.ts (hunk 1 above)
Comment (DeNic0la) [inline functions/src/services/lobby.service.ts:33]: please make this !== (or explain why not)

File: src/app/storify/game/submissions-view/submissions-view.component.scss [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+      margin-top: 44px;
+      transform: scale(0);
+      animation: appear 1000ms;
+      animation-delay: 1400ms;
+      animation-fill-mode: forwards;
+    }
+  }
+
+  .showing {
+  }
```
Comment (DeNic0la) [inline src/app/storify/game/submissions-view/submissions-view.component.scss:59]: (micro) remove

**Review by DeNic0la — CHANGES_REQUESTED** [review]

**Review by Sili-Dev — APPROVED** [review]

**Review by DeNic0la — DISMISSED** [review]

**Review by Sili-Dev — CHANGES_REQUESTED** [review]
I have to fix something

**Review by Sili-Dev — APPROVED** [review]

**Pull request merged**